### PR TITLE
Add conan

### PR DESCRIPTION
**Pull request by Lucas-Vertrees-Bose**

### CODE REVIEW

File: Bose/build-qc8017_32 [hunk 1]
```diff
@@ -5,18 +5,24 @@ set -x
...
   autoconf || exit
 fi
-cd Bose/ || exit
+
+if (( $# != 0 )); then
+    echo >&2 "$this: unexpected argument '$1'"
+    exit 1
+fi
+
+if [ -d "./Bose" ]; then
```
Comment (Michael-Cook-Bose) [inline Bose/build-qc8017_32]: Why would the Bose directory not exist?

File: Bose/build-qc8017_32 [hunk 2]
```diff
@@ -60,9 +65,14 @@ objdump -x $build_dir/lib/libnghttp2.so.* | grep RUNPATH
 
 components record $build_dir || exit
 
-cd "$build_dir/.." || exit
-base=$(basename "$build_dir")
-tar -cjf "$base.tar.bz2" "$base" || exit
-echo "Wrote $PWD/$base.tar.bz2"
+if [ -z "$DONT_TAR" ]
+then
```
Comment (Michael-Cook-Bose) [inline Bose/build-qc8017_32]: `if [ -z "$DONT_TAR" ]; then`

File: Bose/build-qc8017_32 (hunk 1 above)
Comment (Mark-Hatch-Bose) [reply Bose/build-qc8017_32]: My hunch is if this component needs to be rebuilt from source.
If it does a git checkout, we can run the build script from the top level dir without knowledge of the Bose dir.
@Lucas-Vertrees-Bose is that the reasoning here?

Comment (Michael-Cook-Bose) [reply Bose/build-qc8017_32]: Should probably add a comment explaining what's going on here.

Comment (Lucas-Vertrees-Bose) [reply Bose/build-qc8017_32]: Yup @Mark-Hatch-Bose, that was my original reasoning. I can `cd` into the `Bose` folder before running the scripts to get rid of these checks.

File: Bose/build-qc8017_32 [hunk 3]
```diff
@@ -32,7 +38,6 @@ CPP="arm-oemllib32-linux-gcc -E  -fPIC -march=armv7-a -mfloat-abi=softfp -mfpu=n
 CXX="arm-oemllib32-linux-g++  -fPIC -march=armv7-a -mfloat-abi=softfp -mfpu=neon -mtune=cortex-a53 --sysroot=$sysroot"
 LDFLAGS="-Wl,-R,'\$\$ORIGIN/../lib'"
 
-cd "$(dirname "$0")" || exit
```
Comment (Michael-Cook-Bose) [inline Bose/build-qc8017_32]: Perhaps we want to resurrect this line.

**Pull request merged**